buildbot/master/files/config/factories.py

```diff
@@ -187,8 +187,11 @@ def run(self):
         self.is_windows = re.match('windows.*', self.builder_name) is not None
         try:
             show_cmd = "cat" if not self.is_windows else "type"
+            native_yaml_path = self.yaml_path
+            if self.is_windows:
+                native_yaml_path = native_yaml_path.replace('/', '\\')
             cmd = yield self.makeRemoteShellCommand(
-                command=[show_cmd, "{}".format(self.yaml_path)],
+                command=[show_cmd, "{}".format(native_yaml_path)],
```

Comment:
While you're at it, can you get rid of the useless `format` call and just use `native_yaml_path` directly? I missed that on my last PR which I did in a rush.